### PR TITLE
Modified README to improve project documentation

### PR DESCRIPTION
**Pull request by mcarmonaa**
README modifications:
- How to add `siva-java` as a dependency.
- Added example of usage for java.
- Added development section with make commands.

***This PR is related to https://github.com/src-d/siva-java/issues/8***

### CODE REVIEW

**Review by erizocosmico — APPROVED** [review]

**Review by bzz — APPROVED** [review]
LGTM

As we already have #11 - do you think we could post usage instructions in README for maven artefact without jitpack? 

Could also wait to get it published and then update in new PRs (here and in engine)

**Comment by mcarmonaa** [comment]
@bzz I prefer change the README instructions when it will be actually available in maven central in a future PR.

**Comment by erizocosmico** [comment]
Agree with @mcarmonaa about the maven thing

File: README.md [hunk 1]
```diff
@@ -1,19 +1,107 @@
...
 The main implementation is written in Go [here](https://github.com/src-d/go-siva).
 
-## Tests
+This java library offers an API to read and unpack [siva files](https://github.com/src-d/go-siva/blob/master/SPEC.md) but not to write them yet.
 
-Just run `sbt test`
+## Usage
 
-## Installation
+For the moment `siva-java` is available on [jitpack](https://jitpack.io/#src-d/siva-java/). To include it as a dependency in your project managed by [sbt](http://www.scala-sbt.org/), add the resolver and the dependency to your `build.sbt` file:
```
Comment (ajnavarro) [inline README.md]: Now is in maven central, right?

Comment (erizocosmico) [reply README.md]: Now it's on maven

Comment (mcarmonaa) [reply README.md]: I can't find [siva-java on maven](https://search.maven.org/#search%7Cga%7C1%7Csiva-java)

**Review by erizocosmico — APPROVED** [review]

**Comment by mcarmonaa** [comment]
Updated README to use `siva-java` as a dependency from `maven central`

**Review by ajnavarro — APPROVED** [review]

**Review by bzz — APPROVED** [review]

**Review by ajnavarro — APPROVED** [review]

**Pull request merged**